lib/prepareOptions.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function prepareOptions(options, argv) {
```

Comment:
@sokra Thanks, I wasn't sure if non-ES6 was intentional. Updated, though still using functions for top-level rather than const + arrow - let me know if I should change. We should consider added staged-only linting for this.